docs/src/pages/theming/theming.web.mdx

```diff
@@ -106,6 +106,45 @@ const myTheme = {
 };
 ```
 
+#### Defining component tokens
```

Comment:
Rather than being about contributing, shouldn't this be aimed at explaining to users of our theming system?

```suggestion
#### Component token definitions
```

Reply:
Yeah, that suggestion feels more appropriate for the audience.